mixer/cmd/config.go

```diff
@@ -73,6 +73,17 @@ var configSetCmd = &cobra.Command{
 	the existence of the provided property, but will not validate the value provided.`,
 	Args: cobra.ExactArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
+		// Look for the value in mixer.state first
```

Comment:
Based on my feedback for the 'get' command, I think the help message should be updated.